packages/ai/src/workflows/createSquiggleWorkflowTemplate.ts

```diff
@@ -16,10 +16,8 @@ export const createSquiggleWorkflowTemplate = new WorkflowTemplate<{
   outputs: Record<string, never>;
 }>({
   name: "CreateSquiggle",
-  configureControllerLoop(workflow, inputs) {
-    fixAdjustRetryLoop(workflow, inputs.prompt);
-  },
-  configureInitialSteps(workflow, inputs) {
-    workflow.addStep(generateCodeStep, { prompt: inputs.prompt });
-  },
+  getInitialStep: (workflow) =>
+    generateCodeStep.prepare({ prompt: workflow.inputs.prompt }),
```

Comment:
This API is also new: instead of imperative "configure" = "do anything, inject even handlers", now we return the step itself.

I had to implement `PreparedStep` types, separate from both `LLMStepTemplate` and `LLMStepInstance`, and I'm not happy about that, but I think it was unavoidable (see the long comment on `PreparedStep` type in `LLMStepTemplate`).